corehq/apps/registry/templates/registry/registry_edit.html

```diff
@@ -448,7 +452,14 @@ <h4 class="modal-title">{% trans "Grant access" %}</h4>
               {% endblocktrans %}
             </p>
           </div>
-          <select multiple class="form-control" data-bind="select2: availableGrantDomains, selectedOptions: grantDomains"></select>
+          <select multiple class="form-control" data-bind="
+              selectedOptions: grantDomains,
+              multiselect: {
+                selectableHeaderTitle: '{% trans_html_attr "Available Project Spaces" %}',
+                selectedHeaderTitle: '{% trans_html_attr "Selected" %}',
+                searchItemTitle: '{% trans_html_attr "Search" %}',
+                values: availableGrantDomains
```

Comment:
@snopoke This makes the multiselect accept dynamic values, right?

@gherceg FYI, since you've also been working on this widget.

Reply:
yes that's correct, it can now accept values from a knockout observable. I haven't tested adding / removing values dynamically though.